src/components/app/Home.js

```diff
@@ -440,6 +449,45 @@ class HomeImpl extends React.PureComponent<HomeProps, HomeState> {
     );
   }
 
+  _renderFirefoxAndroidInstructions() {
+    return (
+      <InstructionTransition key={0}>
+        <div
+          className="homeInstructions"
+          data-testid="home-firefox-android-instructions"
+        >
+          {/* Grid container: homeInstructions */}
+          {/* Left column: img */}
+          <img
+            className="homeSectionScreenshot"
+            src={PerfScreenshot}
+            alt="screenshot of profiler.firefox.com"
+          />
+          {/* Right column: instructions */}
+          <div>
+            <DocsButton />
+            <Localized id="Home--instructions-title">
+              <h2>How to view and record profiles</h2>
+            </Localized>
+            <Localized
+              id="Home--instructions-content"
+              elems={{
+                a: <a href="https://www.mozilla.org/en-US/firefox/new/" />,
+              }}
```

Comment:
We are using Fluent for localization of strings and we have ftl files for various languages. We should also update our main locale (which is `en-US`), so our community can localize this changed text. This `Localized` component is our helper component that fetches the localized string from our app.ftl file. 

Right now we still use the id of the previous component `Home--instructions-content`. We should also change this id and add a new entry to our app.ftl file with the updated text. This is what needs to be done:
1. Change the id of this component to something like `Home--firefox-android-instructions-content`.
2. Open the `locales/en-US/app.ftl` file and find the previous `Home--instructions-content` we have.
3. Copy and paste that one and add a new `Home--firefox-android-instructions-content` string with the updated text like you have here.

Note that we also have more app.ftl files for different locales but you don't need to update them. Our localizers will update them.